src/dataviews/histogram-dataview/histogram-data-model.js

```diff
@@ -32,6 +33,7 @@ module.exports = Model.extend({
   },
 
   initialize: function () {
+    this.sync = BackboneCancelSync.bind(this);
```

Comment:
Do you think it'd make sense to have this same behaviour in other dataviews (move this to `DataviewModelBase#initialize`)? I think it could be a good thing to avoid race conditions in other widgets too. What do you think?